### PR TITLE
Fix bunch of typos

### PR DESCRIPTION
**Pull request by hqhq**
**\- What I did**

**\- How I did it**

**\- How to verify it**

**\- Description for the changelog**

**\- A picture of a cute animal (not mandatory but encouraged)**

Signed-off-by: Qiang Huang h.huangqiang@huawei.com

### CODE REVIEW

File: integration-cli/docker_api_containers_test.go [hunk 1]
```diff
@@ -957,9 +957,9 @@ func (s *DockerSuite) TestContainerAPIWait(c *check.C) {
 	c.Assert(status, checker.Equals, http.StatusOK)
 	c.Assert(waitInspect(name, "{{ .State.Running  }}", "false", 60*time.Second), checker.IsNil)
 
-	var waitres types.ContainerWaitResponse
-	c.Assert(json.Unmarshal(body, &waitres), checker.IsNil)
-	c.Assert(waitres.StatusCode, checker.Equals, 0)
+	var waiters types.ContainerWaitResponse
```
Comment (thaJeztah) [inline integration-cli/docker_api_containers_test.go]: I think this was meant to be called `waitres` (based on `wait` `res(ponse)`)

File: cli/command/swarm/join_token.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ func newJoinTokenCommand(dockerCli *command.DockerCli) *cobra.Command {
 					return err
 				}
 				if !quiet {
-					fmt.Fprintf(dockerCli.Out(), "Succesfully rotated %s join token.\n\n", args[0])
+					fmt.Fprintf(dockerCli.Out(), "Successfully rotated %s join token.\n\n", args[0])
```
Comment (thaJeztah) [inline cli/command/swarm/join_token.go]: Oh! Think this one was my mistake :blush:

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Thanks @hqhq ! Found a couple that did not have to be changed

**Comment by hqhq** [comment]
@thaJeztah Thanks, updated.

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

/cc @vdemeester :rabbit:

**Comment by duglin** [comment]
LGTM

**Pull request merged**